### PR TITLE
Switch from Travis to Gitlab

### PR DESCRIPTION
**Pull request by amaurymartiny**
cc @ddorgan 

TODO still:
- [x] Add windows build
- [x] Remove travis.yml

### CODE REVIEW

File: gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  untracked: true
+linux-test:
+  stage: test
+  image: node:8
+  only:
+    - beta
+    - tags
+    - stable
+    - triggers
+    - master
```
Comment (amaurymartiny) [inline gitlab-ci.yml]: Will linux-test run on PRs? (it should)

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  untracked: true
+linux-test:
+  stage: test
+  image: node:8
+  only:
+    - beta
+    - tags
+    - stable
+    - triggers
+    - master
```
Comment (amaurymartiny) [inline .gitlab-ci.yml]: Will linux-test run on PRs? (it should)

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  script:
+    - curl -o- -L https://yarnpkg.com/install.sh | bash 
+    - export PATH=$HOME/.yarn/bin:$PATH
+    - yarn install
+    - yarn lint
```
Comment (axelchalon) [inline .gitlab-ci.yml]: I don't think we want to include `yarn lint`. `yarn test` already checks if the files are correctly linted

Comment (ddorgan) [reply .gitlab-ci.yml]: Thanks will remove.

**Comment by axelchalon** [comment]
I think `yarn test` should stay though ^^

**Comment by ddorgan** [comment]
Which mac os x artifact do you want?

  • building        target=macOS zip arch=x64 file=dist/Parity Fether-0.1.0-mac.zip
  • building        target=DMG arch=x64 file=dist/Parity Fether-0.1.0.dmg
  • building block map blockMapFile=dist/Parity Fether-0.1.0.dmg.blockmap

**Comment by amaurymartiny** [comment]
For mac: the .zip and .dmg
For windows: the .exe
For linux: AppImage, snap, deb and tar.xz

**Pull request merged**